ETL_raw_to_trusted_IDEAM.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    df_variables_gen = spark.createDataFrame(df_variables_gen.tail(df_variables_gen.count()-1), df_variables_gen.schema)
</pre><p>Use OFFSET 1 in the query to avoid doing this.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/maestria-ds-eafit/Trabajo1_Almdatos/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>